packages/@react-spectrum/numberfield/src/NumberField.tsx

```diff
@@ -85,7 +85,14 @@ function NumberField(props: SpectrumNumberFieldProps, ref: FocusableRef<HTMLElem
     <Field
       {...props as Omit<SpectrumNumberFieldProps, 'onChange'>}
       labelProps={labelProps}
-      ref={domRef}>
+      ref={domRef}
+      wrapperClassName={classNames(
+        stepperStyle,
+        'spectrum-Stepper-container',
+        {
+          'spectrum-Stepper-container--isMobile': isMobile
+        }
+      )}>
```

Comment:
Could this be done via UNSAFE_className instead? Noticed that Field passes `styleProps.className` to the wrapping div already, just wondering if that should be leveraged instead.